### DISCUSSION TITLE
How to set column width based on content length?

### DISCUSSION THREAD

**Discussion by idafoh** [glideapps/glide-data-grid#340] [Q&A] [answered]
https://github.com/glideapps/glide-data-grid/issues/181#issue-1145416583

**Comment by jassmith** [comment, marked as answer]
your grid columns must have an `id` and not have a `width` set.

**Reply by idafoh** [reply]
<img width="313" alt="Screen Shot 2022-05-23 at 1 35 10 pm" src="https://user-images.githubusercontent.com/83906251/169779045-b5d0bfa0-a410-4160-ab53-03db171e9314.png">
Here is the result I set an id and remove width.

> ------------------
<img width="429" alt="Screen Shot 2022-05-23 at 1 36 37 pm" src="https://user-images.githubusercontent.com/83906251/169779176-17c5c401-9878-4c1c-b9be-0978ef251772.png">
Expected behaviour. (I set width manually here)

**Reply by jassmith** [reply]
Can you file a bug and make a recording with replay.io? This is not expected.

**Reply by idafoh** [reply]
I created [repository](https://github.com/dauletbaev/reproduce-issue-glide-data-grid) with this behaviour.
